packages/api/src/external/carequality/document/process-outbound-document-retrieval-resps.ts

```diff
@@ -29,6 +29,33 @@ export async function processOutboundDocumentRetrievalResps({
     let issuesWithGateway = 0;
     let successDocsCount = 0;
 
+    if (results.length === 0) {
+      const msg = `No results received to process.`;
```

Comment:
Let's be more explicit about what results? Like "Received DR result without entries" or something like that?